bigbluebutton-html5/imports/api/video/server/methods/userUnshareWebcam.js

```diff
@@ -1,10 +1,9 @@
 import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
-import Meeting from '/imports/api/meetings';
 import Logger from '/imports/startup/server/logger';
 import RedisPubSub from '/imports/startup/server/redis';
 
-export default function userUnshareWebcam(credentials, message) {
+export default function userUnshareWebcam(credentials, stream) {
```

Comment:
bigbluebutton-html5/imports/api/video/server/methods/userUnshareWebcam.js
and bigbluebutton-html5/imports/api/video/server/methods/userShareWebcam.js look 99% identical. Do we need both or could just call a combined method + a boolean?

Reply:
We could refactor this in another commit. But I'm not sure if it's worth it, maybe we'll need to put more specific code in either file in the future. But I'll let the rest of the team decide.

Reply:
I don't think it's a good idea. Having separate calls for sharing/unsharing webcams pairs up with the Redis messages, make things a bit more legible and give us more room for any future extension we might wanna make. The methods are similar, yes, but maybe that might ease things up in the future.